cookbooks/fb_hddtemp/metadata.rb

```diff
@@ -0,0 +1,9 @@
+name 'fb_hddtemp'
+maintainer 'Facebook'
+maintainer_email 'noreply@facebook.com'
+license 'BSD'
+description 'Installs/Configures fb_hddtemp'
```

Comment:
small nit-pick here: 's/fb_//g'

I'll merge this next week, but if you can fix this before then I won't have to follow up with an internal diff.